src/eventhubs/azext_eventhub/_help.py

```diff
@@ -0,0 +1,309 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['eventhubs'] = """
+    type: group
+    short-summary: Manage Azure Event Hubs namespace, eventhub, consumergroup and Geo Recovery configuration - Alias
+"""
+
+helps['eventhubs namespace'] = """
+    type: group
+    short-summary: Manage Azure Event Hubs namespace and authorizationrule
+"""
+
+helps['eventhubs eventhub'] = """
+    type: group
+    short-summary: Manage Azure Event Hubs eventhub and authorization-rule
+"""
+
+helps['eventhubs consumergroup'] = """
+    type: group
+    short-summary: Manage Azure Event Hubs consumergroup
+"""
+
+helps['eventhubs georecovery-alias'] = """
+    type: group
+    short-summary: Manage Azure Event Hubs Geo Recovery configuration - Alias
+"""
+
+helps['eventhubs namespace exists'] = """
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: Create a new topic.
+          text: az eventhubs namespace exists --name mynamespace
+"""
+
+helps['eventhubs namespace create'] = """
+    type: command
+    short-summary: Creates the Event Hubs Namespace
+    examples:
+        - name: Create a new namespace.
+          text: az eventhubs namespace create --resource-group myresourcegroup --name mynamespace --location westus
+           --tags tag1=value1 tag2=value2 --sku-name Standard --sku-tier Standard' --is-auto-inflate-enabled False --maximum-throughput-units 30
```

Comment:
There is an extra quote in `Standard'`. This should be removed.

Reply:
resolved